recipes/pyScienceMode/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "pysciencemode" %}
+{% set version = "1.1.4" %}
+{% set python_min = "3.10" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://files.pythonhosted.org/packages/51/a4/14e4d9e859a0109c4e44c36f7e759ae8f3d24a603c72ddf28e9ee9919566/pysciencemode-1.1.4.tar.gz
+  sha256: 91952b59df0ab6b7472c302fd0b5777e271958e57c48d00d063ca1769faa70e6
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
```

```suggestion
  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
```

Consider using `--no-deps --no-build-isolation` to ensure that all of the required build system dependencies are included as `host` dependencies. (I'm not a conda-forge/core maintainer, I'm just a drive by person who happened to be checking to make sure other PRs were failing on macOS. :+1:)

---

@Kev1CO sorry it seems the recommendation is to add it back given https://github.com/conda-forge/staged-recipes/pull/28268#discussion_r1850668020. I'm going to stop commenting on this PR though given that I seem to have given you conflicting advice twice now.